src/DivideIQ.php

```diff
@@ -148,8 +148,8 @@ public function setFile(\SplFileObject $file)
      *
      * @param string $serviceName
      *     The codename of the service to access.
-     * @param array $payload
-     *     (optional) The data to send with the request.
```

Comment:
Why is this? Now the param became specific to the orderrequest, but it's generically applicable.

Make it `array|\JosnSerializable` and then make sure an array is handled correctly in both POST and GET scenarios